tools/sanity_checks.py

```diff
@@ -180,9 +197,9 @@ def check_new_release(self, name: str, info: T.Dict[str, T.Union[T.List[str], st
             else:
                 s = ', '.join(debian_packages)
                 print(f'The following packages could be required: {s}')
-        subprocess.check_call(['meson', 'setup', '_build'] + options)
-        subprocess.check_call(['meson', 'compile', '-C', '_build'])
-        subprocess.check_call(['meson', 'test', '-C', '_build'])
+        subprocess.check_call(['meson', 'setup', builddir] + options)
+        subprocess.check_call(['meson', 'compile', '-C', builddir])
+        subprocess.check_call(['meson', 'test', '-C', builddir])
```

Comment:
This has worked okay for me locally in the past since it passes --wipe instead of using a temporary directory. Curious why this was needed.

Reply:
Note that it still reuse "_build" when you run locally.